stable/traefik/values.yaml

```diff
@@ -423,6 +423,8 @@ accessLogs:
         # Authorization: redact
 rbac:
   enabled: false
+  serviceAccount:
```

Comment:
This should be commented out by default please.

Reply:
Done